test/uk/gov/hmrc/servicedependencies/service/ServiceConfigsServiceSpec.scala

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.servicedependencies.service
+import java.time.LocalDate
+
+import org.joda.time.DateTime
```

Comment:
Seems a bad idea to mix two different APIs for DateTime implementations - I think we should use java.time.LocalDateTime here to ensure consistency.

Reply:
joda DateTime is used everywhere throughout this project. I don't think it makes sense to try and remove joda DateTime as a part of my changes. 

For my changes, I've used `LocalDate`. I could potentially use the date from joda for consistency.